controllers/networkchaos/podnetworkmanager/podnetworkmanager.go

```diff
@@ -83,7 +93,15 @@ func (m *PodNetworkManager) Commit(ctx context.Context) error {
 					pod := v1.Pod{}
 					err = m.Client.Get(ctx, key, &pod)
 					if err != nil {
-						m.Log.Error(err, "error while finding pod", "key", key)
+						m.Log.Info("pod not found", "key", key, "error", err.Error())
+						err = ErrPodNotFound
```

Comment:
the error may not be `not found`, I think it's better to add an adjustment

Reply:
make sense, I think later we can refine it

Reply:
Yes, I agree with you.